source/include/azure_iot_hub_client.h

```diff
@@ -113,6 +117,24 @@ typedef struct AzureIoTHubClientMethodRequest
     uint16_t usMethodNameLength;   /**< The length of the method name. */
 } AzureIoTHubClientMethodRequest_t;
 
+/**
+ * @brief IoT Hub Command Request struct.
+ */
+typedef struct AzureIoTHubClientCommandRequest
```

Comment:
As this is still not public preview... is it ok to just introduce this new concept and remove old concepts? 

@jspaith

Reply:
Yes, we should be 100% all in on PnP concepts.  So any reference to "method" we should have "Command" instead.  Even at public-preview time we can still break stuff, it's only once we hit GA it's a problem.